### PR TITLE
For Content Review - GSQ Features Vocabulary

### PR DESCRIPTION
**Pull request by KellyVance**
Custodian - @mark-gordon-dnrme 
Peer Reviewer 1 - @DavidCrosswellGSQ 
Peer Reviewer 2 - @GSQ-AI 

David, I have uploaded file to GraphDB test. Can you please reboot to push vocab to vocprez and comment with the link once done?
Andrew this vocabulary is the highest level of geological and administrative features. Discuss as needed for further clarification.

### CODE REVIEW

**Comment by DavidCrosswellGSQ** [comment]
@KellyVance Vocprez test restarted

File: vocabularies/gsq-features.ttl [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:placeName a skos:Concept ;
+    skos:definition "Place Names are names given to natural and artificial geospatial features, such as administrative areas, political regions, mountain ranges, rivers, bays etc."@en ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Place name"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:resourceAccumulation a skos:Concept ;
+    skos:definition "Resource accumulations are naturally occurring accumulations or concentrations of minerals (including metals and coal), hydrocarbons, quarrying resources or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore."@en ;
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: As discussed, minerals covers metals, whether native or not.

Georesource accumulations are naturally occurring masses/volumes or concentrations of minerals (including coal), hydrocarbons, quarrying resources (viz. rock, sand, gravel), groundwater or other geological materials that have existent or potential usefulness/value and application.

Do we include groundwater?

[the remark this answers is not in the document]
Comment (GSQ-AI) [reply vocabularies/gsq-features.ttl]: John, I'll be around in a minute to talk to you about this one. In terms of content, I think this is good. In terms of flow, a little bit of wordsmithing between @KellyVance and myself has produced:

"Georesource accumulations are naturally occurring masses, volumes or concentrations of geological materials that have existing or potential economic value. These may include, but are in no way limited to; minerals (including coal), hydrocarbons, quarry resources or groundwater."

[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: "Georesource accumulations are naturally occurring masses, volumes or concentrations of geological materials that have existing or potential economic value. These resources may include, but are not limited to, minerals (including coal), hydrocarbons, quarry resources and groundwater."

File: vocabularies/gsq-features.ttl [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Resource accumulation"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:stratigraphicFeature a skos:Concept ;
+    skos:definition "A Stratigraphic Feature is one of a number of preferably formalised stratigraphic units—lithostratigraphic, chronostratigraphic, biostratigraphic, sequence stratigraphic, magnetostratigraphic, event stratigraphic, etc.—which assist in the description, classification/subdivision and correlation of generally layered sedimentary rocks that are encompassed by the Law of Superposition."@en ;
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: Stratigraphic Features are units of rock and/or sediment that conform to the Law of Superposition. They may be defined by various methods, including Lithostratigraphy, Biostratigraphy, Chronostratigraphy and Sequence Stratigraphy. Contrastingly, Event Stratigraphy identifies synchronous events that are recognised globally in the stratigraphic record.  <<<< The latter needs a tidy up and more research.

[the remark this answers is not in the document]
Comment (GSQ-AI) [reply vocabularies/gsq-features.ttl]: John,

Do we need to define what Event Stratigraphy is _inside_ the definition for Stratigraphic Features? If not, I think that the first two sentences are great:

"Stratigraphic Features are units of rock and/or sediment that conform to the Law of Superposition. They may be defined by various methods, including Lithostratigraphy, Biostratigraphy, Chronostratigraphy and Sequence Stratigraphy."

File: vocabularies/gsq-features.ttl [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        gafeat:permit,
+        gafeat:placeName,
+        gafeat:resourceAccumulation,
+        gafeat:stratigraphicFeature,
+        gafeat:tectonicStructuralFrameworkFeature,
+        gafeat:subBlock ;
+    skos:prefLabel "GSQ Geological and Administrative Features"@en .
+
+gafeat:block a skos:Concept ;
+    skos:definition "A geographic grid used for exploration tenure management using the basic unit (a BLOCK) defined as five minutes of latitude by five minutes of longitude. The Blocks are grouped by 1:1 000 000 sheet areas (the basic map unit for description) referred to as Block Identification Map (BIM). Each BIM contains 3456 blocks with the Blocks numbered between 1 and 3456 inclusive."@en ;
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: Sounds good, but I have slightly modified:

A Block is an administrative division of the State of Queensland that is used for tenure management. A Block is defined as five minutes of latitude by five minutes of longitude. Blocks are grouped by 1:1,000,000 map sheet areas, the names of which provide the containing Blocks with a four letter prefix (e.g. Clermont 1:1,000,000 provides CLER). Each sheet area contains 3456 Blocks, with the Blocks numbered between 1 and 3456 inclusive. A Block is divided into 25 Sub-Blocks.

File: vocabularies/gsq-features.ttl [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    skos:definition "A Tectonic and Structural Framework Feature is one of the myriad tectonic, tectonostratigraphic or structural entities that collectively constitute the Geological Framework of Queensland. Tectonic elements encompass cratons, orogens, sedimentary basins (including troughs and depressions), provinces—generally older basins of indefinite extent/form that have been strongly tectonised and/or metamorphosed, etc.; tectonostratigraphic elements include nappes, klippen, etc.; and structural elements, synclines, anticlines, faults, grabens, half-grabens, horsts, fold zones, unconformities, peneplains, etc."@en ;
+    dct:source "Classification after Kumpulainen, R.A., 2017." ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Tectonic and structural framework feature"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:subBlock a skos:Concept ;
+    skos:definition "A geographic grid used for exploration tenure management using the basic unit (a SUB-BLOCK) defined as 1 minute of latitude by 1 minute of longitude. Each Block is divided into 25 sub-blocks. The sub-blocks are labelled between A and Z, excluding I."@en ;
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: Excellent

File: vocabularies/gsq-features.ttl [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    dct:source "NACSN, 1983, 2005; Kumpulainen, R.A., 2017." ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Lithodemic feature"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:permit a skos:Concept ;
+    skos:definition "In general, mineral or energy rights in Australia are reserved to the Crown. The acquisition of rights to minerals or energy stems from legislative frameworks in Queensland. These frameworks provide initially for exploration of the resource, and consist of the grant by the Crown in the form of exploration permits, licences or leases. Exploration permits, leases or licences permit works to be undertaken to determine the likely existence of minerals or resources. Actual mining is subject to a further grant of mining or minerals production leases or licences. The legislation also provides for the payment of royalties to the State and to compensate the owners or occupiers of the surface land."@en ;
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: No, it's good,

File: vocabularies/gsq-features.ttl (hunk 2 above)
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-features.ttl]: Andrew, as per email to Nick

[the remark this answers is not in the document]
Comment (GSQ-AI) [reply vocabularies/gsq-features.ttl]: For the record, the e-mail to Nick suggested:
"Stratigraphic Features embrace Stratigraphic Units, which are units of rock and/or sediment that conform to the Law of Superposition (e.g. lithostratigraphic units, biostratigraphic units, chronostratigraphic units), and Stratigraphic Events which reflect physical, chemical and biological events that left identifiable, widespread, if not worldwide, traces or markers in the sedimentary or extrusive igneous record (ICS; Kumpulainen, R.A., 2016)."

I'd make a few minor edits along the lines of:
"Stratigraphic Features **include** Stratigraphic Units, which are units of rock and/or sediment that conform to the Law of Superposition (e.g. lithostratigraphic units, biostratigraphic units, chronostratigraphic units), and Stratigraphic Events **which are,** chemical and biological events that left identifiable, widespread, if not worldwide, traces or markers in the sedimentary or extrusive igneous record."
dct:source "ICS; Kumpulainen, R.A., 2016"

File: vocabularies/gsq-features.ttl [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    dct:source "NACSN, 1983, 2005; Kumpulainen, R.A., 2017; ICS." ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Stratigraphic feature"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:tectonicStructuralFrameworkFeature a skos:Concept ;
+    skos:definition "A Tectonic and Structural Framework Feature is one of the myriad tectonic, tectonostratigraphic or structural entities that collectively constitute the Geological Framework of Queensland. Tectonic elements encompass cratons, orogens, sedimentary basins (including troughs and depressions), provinces—generally older basins of indefinite extent/form that have been strongly tectonised and/or metamorphosed, etc.; tectonostratigraphic elements include nappes, klippen, etc.; and structural elements, synclines, anticlines, faults, grabens, half-grabens, horsts, fold zones, unconformities, peneplains, etc."@en ;
```
[the remark this answers is not in the document]
Comment (GSQ-AI) [reply vocabularies/gsq-features.ttl]: John's suggestion via e-mail:
"A Tectonic and Structural Framework Feature is a Tectonic, Tectonostratigraphic or Structural element. Tectonic Elements include cratons, orogens, provinces and sedimentary basins; Tectonostratigraphic Elements include nappes and klippen; and Structural Elements include synclines, anticlines, faults and half-grabens."

File: vocabularies/gsq-features.ttl (hunk 2 above)
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-features.ttl]: I'd say "... and Stratigraphic Markers that represent chemical and biological events that left widespread traces or features in the sedimentary or extrusive igneous record." Events are actions (verbs), that result in markers,features,traces (nouns).

[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-features.ttl]: I can live with any of the definitions detailed in these comments. Minor style changes need not prevent us progressing this.
 @johnmckellar can you please update the file in this pull request as needed? Then GSQ-AI and I can mark our comments as resolved and approve the pull request.

**Comment by johnmckellar** [comment]
Mark,
Can you please review for finalisation of this top level of features.

**Comment by johnmckellar** [comment]
Cheers

**Pull request merged**